tests/stress/continuous_integration

```diff
@@ -26,6 +26,11 @@ SMALL_DATASET = [
         "options": ["--vertex-count", "40000", "--create-pack-size", "100"],
         "timeout": 5,
     },
+    {
+        "test": "parser.cpp",
+        "options": ["--client-count", "12", "--per-client-query-count", "1000"],
+        "timeout": 5,
+    },
```

Comment:
This is the only change in this file, everything else was the pre-commit `black` hook